src/apps/Organization/Brand.js

```diff
@@ -0,0 +1,167 @@
+
+  return (
+    <Box padding={3 * GU} heading="Brand">
+      <div css="display: flex; width: 100%">
+        <div css="display: flex; flex-direction: column; width: 50%; padding-right: 12px">
+          <Label text="Logo" />
+
+          <div css="margin-bottom: 20px">
+            <Dropzone onDrop={acceptedFiles => console.log(acceptedFiles)}>
+              {({ getRootProps, getInputProps, isDragActive }) => (
+                <div {...getRootProps()} css="outline: none">
+                  <input {...getInputProps()} />
+                  <div
+                    css={`
+                      background: ${theme.surfaceUnder};
+                      width: 217px;
+                      height: 217px;
+                      padding: 30px;
+                      margin-bottom: 10px;
+                      border: ${isDragActive
+                        ? '1px dashed green'
+                        : '1px solid white'};
+                    `}
+                  >
+                    <img
+                      css={`
+                        width: 157px;
+                        height: 157px;
+                        border: 0;
+                        border-radius: 50%;
+                      `}
+                      src={organizationLogoPlaceholder}
+                      alt=""
+                    />
+                  </div>
+                  <Button
+                    mode="outline"
+                    css="margin-right: 10px; font-weight: bold"
+                  >
+                    Upload new
+                  </Button>
+                  <Button mode="outline" css="font-weight: bold">
+                    Revert to default
+                  </Button>
+                  <Text css="display: block; margin-top: 8px" size="small">
+                    Please keep 1:1 ratio
+                  </Text>
+                </div>
+              )}
+            </Dropzone>
+          </div>
+
+          <Label text="Accent color hex" block>
+            <TextInput
+              css={`
+                display: block;
+                border: 1px solid ${colorError ? 'red' : '#DDE4E9'};
+              `}
+              value={accentColor}
+              onChange={changeAccentColor}
+            />
+          </Label>
+
+          {colorError && (
+            <Text css="margin-top: 3px" color="#F22" size="xsmall">
+              Please use #123 or #123456 format
+            </Text>
+          )}
+
+          <Info css="margin: 20px 0">
+            When empty, the accent color sets itself to the default. Try adding
+            a custom hex value and see the change reflected on the button below
+            as a preview.
+          </Info>
+        </div>
+
+        <div css="display: flex; flex-direction: column; width: 50%; padding-left: 12px">
+          <Label text="Placeholder image" />
+
+          <div css="margin-bottom: 20px">
+            <Dropzone onDrop={acceptedFiles => console.log(acceptedFiles)}>
```

Comment:
I think it would increase the visibility of this placeholder function if we move it up next to the `saveColors` definition.